app/src/main/java/com/gruppo3/wetravel/mapManager/types/DestinationMarker.java

```diff
@@ -0,0 +1,81 @@
+package com.gruppo3.wetravel.mapManager.types;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.gms.maps.model.LatLng;
+
+/**
+ * DestinationMarker object containing the information on a map-displayable marker.
+ *
+ * @author Giovanni Barca
+ */
+public class DestinationMarker {
+    private LatLng latLng = null;
+    private String title = null;
+    private float color;
+    private Object object = null;
+
+    /**
+     * @param latLng Coordinates where to place the marker
+     * @param title String to show when user clicks on the marker (it's showed in a popup over the marker)
+     */
+    public DestinationMarker(@NonNull LatLng latLng, @NonNull String title) {
```

Comment:
If parameters can't be null, wouldn't it be better if you wrote it out in the specifications?
```suggestion
     * @param latLng Coordinates where to place the marker. Never null.
     * @param title String to show when user clicks on the marker (it's showed in a popup over the marker). Never null.
     */
    public DestinationMarker(@NonNull LatLng latLng, @NonNull String title) {
```